.github/workflows/README.md

```diff
@@ -0,0 +1,24 @@
+This directory contains GitHub Actions (GHA) yaml files as well as helper files they use.
+
+## Building and pushing Docker images
+
+* Each workflow `docker_{module name}.yml` is used to build the given module's Docker image.
+* `docker_all-modules.yml` builds and pushes all module Docker images by calling each `docker_{module name}.yml` workflow.
+* The `build-push-docker-module.yml` workflow is not standlone but contains actions used by `docker_all-modules.yml` and each `docker_{module name}.yml`.
+
+## Module testing
+
+* Each workflow `run_{module name}.yml` is used to run the given module in CI using the test data.
+* `run_all-modules.yml` runs all modules in CI by calling each `run_{module name}.yml` workflow.
```

Comment:
I would put this first, as we want to do this for all modules, but Docker comes later.